README.md

```diff
@@ -661,6 +661,29 @@ iperf:
         - *vm1
 ```
 
+SPECIFYING FLAGS IN CONFIGURATION FILES
+=================
+You can now specify flags in configuration files by using the `flags` key at the
+top level in a benchmark config. The expected value is a dictionary mapping
+flag names to their new default values. The flags are only defaults; it's still
+possible to override them via the command line. It's even possible to specify
+conflicting values of the same flag in different benchmarks:
+
+```
+iperf:
+  flags:
+    machine_type: n1-standard-2
+    zone: us-central1-b
```

Comment:
`zone` is the correct flag now after the config PR. If you think we need to keep the `zones` flag, I can put it back in, but it feels like kind of a hack - especially now that there's a better way (assigning zones to vm groups rather than individual vms)